src/main/java/us/ihmc/mecano/algorithms/JointTorqueRegressorCalculator.java

```diff
@@ -279,6 +391,40 @@ public DMatrixRMaj getParameterVector()
       return parameterVector;
    }
 
+   /**
+    * Gets the 10-by-1 slice of the inertial parameter vector corresponding to {@code body}.
+    * <p>
+    * In general, this should mostly be used for debugging purposes, and if called, should be called soon after the
+    * constructor, else the result wil be meaningless (e.g. set to all zeros, or all zeros with a few ones).
+    * </p>
+    * <p>
+    * For more details, see {@link JointTorqueRegressorRecursionStep#parameterVector}.
+    * </p>
+    *
+    * @param body the rigid body to get the inertial parameter vector slice for.
+    * @return the 10-by-1 vector of inertial parameters.
+    */
+   public DMatrixRMaj getParameterVectorSlice(RigidBodyReadOnly body)
+   {
+      return rigidBodyToRecursionStepMap.get(body).parameterVector;
```

Comment:
Let's add some null check like in: https://github.com/ihmcrobotics/mecano/blob/develop/src/main/java/us/ihmc/mecano/algorithms/InverseDynamicsCalculator.java#L578-L584

Reply:
Done a null check on the recursion step, but shouldn't this in my case be a null check on the inertia of the rigid body? I'm unsure. Also, I tried to add a similar null check to `getParameter` below, but that returns a double, so I can only return a `Double.NaN` in that case. I'm not sure that's good?